pkgs/applications/networking/cluster/terraform-providers/lxd/default.nix

```diff
@@ -0,0 +1,26 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "terraform-provider-lxd-unstable-${version}";
+  version = "2019-04-03";
```

Comment:
Why not the stable v1.1.3 ?

Reply:
v1.1.3 would not build for me citing `go/src/github.com/sl1pm4t/terraform-provider-lxd/lxd/resource_lxd_snapshot.go:128:29: snap.CreationDate undefined (type *api.ContainerSnapshot has no field or method CreationDate)`